src/main/resources/sprache/English.properties

```diff
@@ -818,8 +818,10 @@ scout_price=Price
 scout_injury=Injured %weeks% weeks
 scout_status=Status
 scout_success=Parsing successful!
-scout_warning=Possible parse error! Please check values!
-scout_error=Parse error! Please report this player!
+scout_warning=Parsing has failed for the following terms:
+scout_error=Provided data could not be parsed, please check the source.
+scout_error_input_empty=Please insert player data.
+bug_ticket=You can report this by opening a new bug ticket on GitHub
```

Comment:
[ ] Please rebase over latest master and use the definition of bug ticket already introduced otherwise it will confuse translators

Reply:
I think I did well, Could you check?